.github/workflows/dapr.yml

```diff
@@ -226,6 +226,14 @@ jobs:
           done
       - name: generate checksum files
         run: cd ${ARTIFACT_DIR} && for i in *; do sha256sum -b $i > "$i.sha256"; done && cd -
+      - name: import GPG key
+        id: import_gpg
+        uses: crazy-max/ghaction-import-gpg@v3
+        with:
+          gpg-private-key: ${{ secrets.GPG_SECRET_KEY }}
+          passphrase: ${{ secrets.GPG_SECRET_KEY_PASSWORD }}
+      - name: generate signed files
+        run: cd ${ARTIFACT_DIR} && for i in *; do gpg --armor --detach-sign $i; done && cd -
```

Comment:
I don't think we want to sign the checksum files (IMO). Can you change the loop to ignore all `.sha256` files? Otherwise, LGTM.

Reply:
Sure.

For quick refer:
```
root:[dapr]$ ls --help | grep \I
Usage: ls [OPTION]... [FILE]...
List information about the FILEs (the current directory by default).
      --block-size=SIZE      with -l, scale sizes by SIZE when printing them;
                               e.g., '--block-size=M'; see SIZE format below
  -I, --ignore=PATTERN       do not list implied entries matching shell PATTERN
```